### DISCUSSION TITLE
Automatically remove recordings

### DISCUSSION THREAD

**Discussion by petricf** [Motion-Project/motionplus#193] [Q&A]
Is there a configuration option or tool to remove recordings based on age and/or remaining space on disk ?
It should also remove the corresponding entries in the database.

**Comment by Mr-Dave** [comment]
Not yet, but it is on the [list](https://github.com/Motion-Project/motionplus/discussions/56) .

**Comment by Mr-Dave** [comment]
This has now been pushed into the master branch.
